ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/StandaloneExecutorTest.java

```diff
@@ -16,76 +16,229 @@
 
 package io.confluent.ksql.rest.server;
 
+import static org.easymock.EasyMock.anyObject;
 import static org.easymock.EasyMock.anyString;
 import static org.easymock.EasyMock.eq;
 
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
 import io.confluent.ksql.KsqlEngine;
 import io.confluent.ksql.function.UdfLoader;
+import io.confluent.ksql.metastore.MetaStore;
+import io.confluent.ksql.parser.tree.CreateStream;
+import io.confluent.ksql.parser.tree.CreateStreamAsSelect;
+import io.confluent.ksql.parser.tree.CreateTable;
+import io.confluent.ksql.parser.tree.CreateTableAsSelect;
+import io.confluent.ksql.parser.tree.DropStream;
+import io.confluent.ksql.parser.tree.InsertInto;
+import io.confluent.ksql.parser.tree.QualifiedName;
+import io.confluent.ksql.parser.tree.Query;
+import io.confluent.ksql.parser.tree.SetProperty;
+import io.confluent.ksql.parser.tree.UnsetProperty;
+import io.confluent.ksql.serde.DataSource.DataSourceType;
 import io.confluent.ksql.util.KsqlConfig;
+import io.confluent.ksql.util.KsqlException;
+import io.confluent.ksql.util.Pair;
 import io.confluent.ksql.util.PersistentQueryMetadata;
 import io.confluent.ksql.util.QueryMetadata;
-import java.io.FileOutputStream;
 import java.io.IOException;
-import java.nio.charset.StandardCharsets;
 import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
 import org.apache.kafka.test.TestUtils;
 import org.easymock.EasyMock;
 import org.junit.Before;
 import org.junit.Test;
 
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
 public class StandaloneExecutorTest {
 
   private final KsqlConfig ksqlConfig = new KsqlConfig(Collections.emptyMap());
   private final KsqlEngine engine = EasyMock.niceMock(KsqlEngine.class);
   private final UdfLoader udfLoader = EasyMock.niceMock(UdfLoader.class);
-  private final String query = "select * from bar;";
-  private StandaloneExecutor executor;
+  private String queriesFile;
+  private StandaloneExecutor standaloneExecutor;
+  private MetaStore metaStore;
+  private final Map<String, Object> props = ImmutableMap.of();
+  private final QualifiedName qualifiedName = QualifiedName.of("Test");
+
+  final Query query = EasyMock.niceMock(Query.class);
```

Comment:
mocks should not really be created inline. Ideally use the `@Mock(type = Nice)` annotation and appropriate class runner, or else initialize in `before()`

Reply:
Moved them to `before()`.

Reply:
> Moved them to before().

They haven't moved. 

You could also give the annotations a go... go on ;)

```
@RunWith(EasyMockRunner.class)
public class StandaloneExecutorTest {

  ...

  @Mock(MockType.NICE)
  private Query query;
  @Mock(MockType.NICE)
  private QueryMetadata persistentQueryMetadata;

  ...
```